src/deadline_worker_agent/sessions/session.py

```diff
@@ -299,6 +311,7 @@ def run(self) -> None:
             # service. If an action was running at the time of this exception, its failure is
             # reported immediately in the call to Session._cleanup() below.
             self._stop_fail_message = f"Worker encountered an unexpected error: {e}"
+            self.logger.info(self._stop_fail_message)
```

Comment:
this should be logged as an error:


```suggestion
            self.logger.error(self._stop_fail_message)
```